Makefile

```diff
@@ -4,7 +4,7 @@ NAMESPACE=pmuir
 NAME=rhoas
 BINARY=terraform-provider-rhoas
 VERSION ?= 0.1
-OS_ARCH ?= darwin_amd64
+OS_ARCH ?= darwin_arm64
```

Comment:
Not sure if we have this right

Reply:
only way to get it to run on my machine, ideally we would have a check to figure out what os and architecture you are on

Reply:
https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script

Reply:
Let's make things right. Changing this value to other value will invalidate it for others etc.